src/graphql/data/RelayEnvironment.ts

```diff
@@ -1,16 +1,38 @@
 import ms from 'ms.macro'
-import { Environment, Network, RecordSource, Store } from 'relay-runtime'
-
-import fetchGraphQL from './fetchGraphQL'
```

Comment:
Is this file still used? I thought it had other semantics that aren't duplicated here.

Reply:
the new library does all of this for us. This is what fetchGraphql was:

```
const fetchQuery = (params: RequestParameters, variables: Variables): Promise<GraphQLResponse> => {
  const body = JSON.stringify({
    query: params.text, // GraphQL text from input
    variables,
  })

  return fetch(GRAPHQL_URL, { method: 'POST', body, headers })
    .then((res) => res.json())
    .catch((e) => {
      console.error(e)
      return { data: [] }
    })
}
```

Reply:
big

Reply:
Can you delete the old file?

Reply:
It already was. Might've accidentally added it back in between pushes maybe